lib/go-tc/statuses.go

```diff
@@ -65,7 +65,7 @@ type Status struct {
 // StatusNullable is a nullable single Status response for Update and Create to
 // depict what changed.
 type StatusNullable struct {
-	Description *string    `json:"description"`
+	Description *string    `json:"description" db:"description"`
```

Comment:
Were these changes to Statuses and their handler(s) in TO necessary consequences of upgrading dependencies? Or just something you did because you noticed it?

Reply:
Yes, this was necessary. Without it, the v4, v3, and v2 API tests failed with

```go
 --- FAIL: TestStatuses (0.30s)
    statuses_test.go:210: results do not match actual: OFFLINE, expected: new description
```
The reason is linked in the commit message of f3d809c087:

> Remove null Status description workaround from apache#3148 now that jmoiron/sqlx#429 is merged